src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -136,6 +137,19 @@ async Task<string> IDurableOrchestrationClient.StartNewAsync(string orchestrator
             return instance.InstanceId;
         }
 
+        private bool TryGetStatusesNotToOverride(out OrchestrationStatus[] dedupeStatuses)
```

Comment:
I wouldn't follow the `Try*` pattern unless you are actually going to check if the request succeeded or not.